Ansible/todoapp.service

```
@@ -0,0 +1,6 @@
+[Unit]
+Description=My To-Do App
+[Service]
+User=ec2-user
+WorkingDirectory=/opt/todo
+ExecStart=/home/ec2-user/.local/bin/poetry run flask run -h 0.0.0.0 --server.port=5000
```

The CLI option is "port", not "server.port".
```suggestion
ExecStart=/home/ec2-user/.local/bin/poetry run flask run -h 0.0.0.0 --port=5000
```

But port 5000 is the default so you could actually skip that setting altogether.